client/src/components/Messaging/Message/ChartMessage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Typography } from '@mui/material';
+import { useStyles } from './useStyles';
+
+interface props {
+  sender: string;
+  message: string;
+}
+const ChatMessage = ({ sender, message }: props): JSX.Element => {
+  const classes = useStyles();
+  const messageClass = sender === 'me' ? classes.sent : classes.received;
+
+  const textStyle = messageClass.split('-')[1] === 'sent' ? classes.textSent : classes.textRecieved;
+
+  return (
+    <>
+      <div className={`message ${messageClass}`}>
```

Comment:
Let's use MUI `Box` component to replace this `div`

Reply:
Please address this comment Rashid 🙂